assets/scss/base/layouts/_layouts.scss

```diff
@@ -96,3 +96,47 @@ body {
 .row {
   margin-bottom: 20px;
 }
+
+/*
+Half column container component
+
+Experimental: Half column component
+
+Markup:
+<div class="column-half-container">
+    <div class="grid-wrapper row-border">
+        <div class="grid grid-1-2">
+            <div>Some content</div>
+        </div>
+        <div class="grid grid-1-2">
+            <div>Some content</a>
+        </div>
+    </div>
+</div>
+
+.column-half-container - component containing two half column grids redefining parts of row-border and grid-wrapper elements
+*/
+.column-half-container {
+  margin-bottom: em(25);
+
+  .row-border {
+    border-bottom: 1px solid $light-grey;
+    margin: 0;
+    padding: em(14.25) 0 em(10.68) 0;
+  }
+
+  .grid-wrapper p {
+    margin: 0.25em 0;
+  }
+}
+
+@media(max-width: 966px){
+  .column-half-container  {
+    .grid.grid-1-2 {
```

Comment:
@Jagordon the options in `assets-frontend` are to add `.grid-layout--stacked` which will stack at mobile or to add your own modifier in grids that keeps 50/50 grid for all displays